stories/query-rule-custom-data.stories.ts

```diff
@@ -37,7 +37,7 @@ storiesOf('QueryRuleCustomData', module)
                   const { title, banner, link } = item;
 
                   if (!banner) {
-                    return;
+                    return undefined;
```

Comment:
We can return the same data type.

```suggestion
                    return '';
```